hack/build-metrics-exporter.sh

```diff
@@ -5,4 +5,6 @@ set -e
 source hack/common.sh
 source hack/docker-common.sh
 
-${IMAGE_BUILD_CMD} build --build-arg="LDFLAGS=${LDFLAGS}" --no-cache -f metrics/Dockerfile -t "${METRICS_EXPORTER_FULL_IMAGE_NAME}" .
+${IMAGE_BUILD_CMD} build  --no-cache \
+    --build-arg="GOOS=${GOOS}" --build-arg="GOARCH=${GOARCH}" --build-arg="LDFLAGS=${LDFLAGS}" \
```

Comment:
Can we move the build args in the last line, Pls check if it works for both docker and podman